lib/stdlib/src/math.erl

```diff
@@ -154,5 +154,7 @@ tanh(_) ->
 %%% End of BIFs
 
 -spec pi() -> float().
-
 pi() -> 3.1415926535897932.
+
+-spec tau() -> float().
+tau() -> 6.2831853071795864.
```

Comment:
Just a suggestion. Instead of writing out the full (yeah, I know) sequence of digits, I would put the value for pi in a constant and define both `pi/0` and `tau/0` in terms of it:
```erlang
-define(PI, 3.1415926535897932).

pi() -> ?PI.

tau() -> 2 * ?PI.
```
You know, in case some day god decides to [make math things simpler](https://www.smbc-comics.com/comic/irrational-3) :smile_cat:

Reply:
I didn't wanted to do it to avoid loosing precision when compiling this module on different machines. We can make math simpler, but IEEE 754 will always stand in our way.